### PR TITLE
Enable torch_xla whl build with python 3.9

### PR DESCRIPTION
**Pull request by yeounoh**
* Set default build Python ver 3.6 -> 3.8
* Bug fixes for Python 3.9 whl builds (tbb package was blocking the build)

### CODE REVIEW

**Comment by yeounoh** [comment]
Py3.9 passed, https://pantheon.corp.google.com/cloud-build/builds;region=us-central1/597ad7a1-d43b-43af-8938-fd4b01e2fe8b?project=tpu-pytorch

File: scripts/build_torch_wheels.sh [hunk 1]
```diff
@@ -6,7 +6,7 @@ set -x  # Display commands being run.
 PYTHON_VERSION=$1
 RELEASE_VERSION=$2  # rX.Y or nightly
 BUILD_CPP_TESTS="${3:-0}"
-DEFAULT_PYTHON_VERSION=3.6
+DEFAULT_PYTHON_VERSION=3.8
```
Comment (alanwaketan) [inline scripts/build_torch_wheels.sh:9]: Is this supposed to be 3.9?

File: scripts/build_torch_wheels.sh [hunk 2]
```diff
@@ -202,6 +202,7 @@ function install_and_setup_conda() {
   conda install -y numpy pyyaml setuptools cmake cffi typing tqdm coverage tensorboard hypothesis dataclasses
   if [[ $(uname -m) == "x86_64" ]]; then
     # Overwrite mkl packages here, since nomkl conflicts with the anaconda env setup.
+    conda remove -y tbb
```
Comment (alanwaketan) [inline scripts/build_torch_wheels.sh:205]: What's this?

File: scripts/build_torch_wheels.sh (hunk 1 above)
Comment (alanwaketan) [reply scripts/build_torch_wheels.sh:9]: Missed the PR description. So what I guess my real question is what makes this patch support python 3.9?

Comment (yeounoh) [reply scripts/build_torch_wheels.sh:9]: The default is 3.8, and on GCB we can set python version per trigger.

File: scripts/build_torch_wheels.sh (hunk 2 above)
Comment (yeounoh) [reply scripts/build_torch_wheels.sh:205]: This is what supports python 3.9 wheel.

**Review by alanwaketan — APPROVED** [review]
LGTM.

File: scripts/build_torch_wheels.sh (hunk 2 above)
Comment (vanbasten23) [reply scripts/build_torch_wheels.sh:205]: can you explain more? Why python <=3.8 don't need this?

Comment (yeounoh) [reply scripts/build_torch_wheels.sh:205]: Not sure why, but this one was blocking the build when used with Python 3.9.

**Pull request merged**